src/doctopus/template.clj

```diff
@@ -1,29 +1,29 @@
 (ns doctopus.template
   (:require [doctopus.doctopus :refer [list-heads list-tentacles]]
-   [net.cgrand.enlive-html :as enlive :refer [deftemplate defsnippet]]
+            [net.cgrand.enlive-html :as enlive :refer [deftemplate defsnippet]]
             [doctopus.configuration :refer [server-config]]
             [doctopus.doctopus :refer [list-heads list-tentacles list-tentacles-by-head]]
             [doctopus.doctopus.tentacle :refer [get-html-entrypoint]]
             [ring.util.anti-forgery :as csrf]))
 
-(defn- make-anchor
```

Comment:
If you ever need/want to, you *can* actually reference private symbols in a test namespace -- something like `(#'namespace/private-fn)`.